WordPress/gutenberg
# Align checkbox, radio, and toggle input design

<img width="893" alt="storybook" src="https://github.com/user-attachments/assets/e82bd8a7-7b54-4aa8-b3b3-afa3484bcb08">

This PR is to scrutinize some design system explorations about better aligning `ToggleControl`, `RadioControl`, and `CheckboxControl`.

### Changes

* Align labels
  * Line-height matches input height to ensure alignment
  * `cursor: pointer` across
* Reduced footprint for Checkboxes and Radios on larger screens making them more proportionally aligned with the label
* Update toggle dimensions based on 4px baseline. 16px height to align with radio + checkbox.
* Apply the standard input border color to toggle off state
* Indent help text consistently between checkbox and toggle (aligned with label on Y axis)
* Consistent focus ring styling
* Consistent spacing between input and help text
* Consistent spacing between input and label

### To test

* `npm run storybook:dev`
* Check `RadioControl`, `CheckboxControl`, `ToggleControl`
* Smoke test the Editor

### Changes in context
<img width="1873" alt="inputs" src="https://github.com/user-attachments/assets/428c64a4-4b00-4806-be64-594fe2f8c51b">
